src/components/CollectionsAndPalettes/Collections/Collections.tsx

```diff
@@ -24,6 +71,24 @@ export default class Collections extends React.Component<any, any> {
         <Link to={COLLECTIONS_IMAGE_URL}>Image</Link>
         <Link to={COLLECTIONS_INFO_URL}>Info</Link>
         <Link to={COLLECTIONS_EDIT_URL}>Edit</Link>
+        {/*for demo purposes*/}
+        <div
+          style={{
+            display: "grid",
+            gridTemplateColumns: "33% 33% 33%",
```

Comment:
You can use `repeat(3, 1fr)` here instead of `33% 33% 33%`. We should try to avoid using percentages as they won't account for any column gap. Also, i'd make this have 4 columns with 20px column gap as in the design and make the cards themselves not be fixed at 295px but instead extend to fill the column width (I think using width: 100% on the card, imageContainer and img would achieve this)